### PR TITLE
Workspaces tests

### PR DESCRIPTION
**Pull request by idea404**
This PR implements workspaces testing into the `integration-tests` folder (both Rust and TypeScript)

To run: 

*Unit tests*
`cd ft && cargo test -- --nocapture --color=always && cd ..`
*Run Rust integration tests*
`cd integration-tests/rs && cargo run --example integration-tests && cd ..`
*Run TS integration tests*
`cd integration-tests/ts && yarn && yarn test && cd ..`

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable-2020-10-08
+          target: wasm32-unknown-unknown
+      - name: Run unit tests
+        run: cd scripts && chmod +x unit_tests.sh && ./unit_tests.sh
```
Comment (volovyk-s) [inline .github/workflows/tests.yml:25]: You should be able to `chmod` a script on your PC and push this change.

File: integration-tests/ts/src/main.ava.ts [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+ * - Complex transactions: to exercise certain edge cases of the FT standard,
+ *   tests below initiate chains of transactions using near-workspaces's transaction
+ *   builder. Search for `batch` below.
+ */
+import { Worker, NearAccount, captureError, NEAR, BN } from 'near-workspaces';
+import anyTest, { TestFn } from 'ava';
+
+const STORAGE_BYTE_COST = '1.5 mN';
+
+async function init_ft(
```
Comment (volovyk-s) [inline integration-tests/ts/src/main.ava.ts:21]: Why are you using these helper functions? You can just move this logic to `beforeEach`.

File: scripts/integration_tests.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (volovyk-s) [inline scripts/integration_tests.sh:1]: I would recommend not to have this kind of script, separate JS and RS jobs on the CI and just using ava (as a yarn test in package.json) and cargo to run them.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (idea404) [reply .github/workflows/tests.yml]: Have moved scripts to the yaml file will update README and PR description accordingly. Thank you!

File: integration-tests/ts/src/main.ava.ts (hunk 2 above)
Comment (idea404) [reply integration-tests/ts/src/main.ava.ts]: Have adjusted in most recent commit. Note I sourced this code from: https://github.com/near/workspaces-js/blob/d7c4f78b6b564082ee708722dd519ab7e9cb81da/__tests__/04.cross-contract-calls-with-fungible-token.ava.ts

So that repo would also have to be updated.
